module/zfs/vdev_raidz.c

```diff
@@ -1440,6 +1440,94 @@ vdev_raidz_child_done(zio_t *zio)
 	rc->rc_skipped = 0;
 }
 
+
+/* Rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_rotating_inc;
+extern int zfs_vdev_mirror_rotating_seek_inc;
+extern int zfs_vdev_mirror_rotating_seek_offset;
+
+/* Non-rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_non_rotating_inc;
+extern int zfs_vdev_mirror_non_rotating_seek_inc;
```

Comment:
Yea, this is definitely not the right place for these. I wanted to keep the changes contained in the first iteration.
These tunables are kinda out of place in the raidz code, but since this is a sort of mirroring situation, I think they could be used here as well. Maybe doc would need updating to reflect this.